store/src/java/com/zimbra/soap/ProxyTarget.java

```diff
@@ -146,17 +146,19 @@ public Element dispatch(Element request, ZimbraSoapContext zsc) throws ServiceEx
                 transport.setTimeout((int) Math.min(mTimeout, Integer.MAX_VALUE));
 
             transport.setResponseProtocol(zsc.getResponseProtocol());
-            AuthToken authToken = AuthToken.getCsrfUnsecuredAuthToken(zsc.getAuthToken());
-            if (authToken != null && !StringUtil.isNullOrEmpty(authToken.getProxyAuthToken())) {
-                transport.setAuthToken(authToken.getProxyAuthToken());
+            if (!zsc.getAuthToken().isJWT()) {
```

Comment:
The previous code implied that perhaps `zcs.getAuthToken()` could return null.  Needs handling if that is possible.